### PR TITLE
Settings + some list fixes

### PR DESCRIPTION
**Pull request by SMJSGaming**

### CODE REVIEW

File: loader/include/Geode/ui/General.hpp [hunk 1]
```diff
@@ -86,7 +86,7 @@ namespace geode {
     public:
         static ListBorders* create();
 
-        void setSpriteFrames(const char* topAndBottom, const char* sides, float topPadding = 7.5f);
+        void setSpriteFrames(const char* topAndBottom, const char* sides, float horizontalPadding = 7.5f);
```
Comment (matcool) [inline loader/include/Geode/ui/General.hpp:89]: why the rename?

Comment (SMJSGaming) [reply loader/include/Geode/ui/General.hpp:89]: look into how it's used
it's used as both top and bottom padding and ends up being used as horizontal padding for the top and bottom sprite

File: loader/src/ui/nodes/TextInput.cpp [hunk 2]
```diff
@@ -151,7 +151,6 @@ void TextInput::setCallback(std::function<void(std::string const&)> onInput) {
     m_onInput = onInput;
 }
 void TextInput::setEnabled(bool enabled) {
-    m_input->setMouseEnabled(enabled);
```
Comment (SMJSGaming) [inline loader/src/ui/nodes/TextInput.cpp:154]: To clarify, this caused the scroll issues in all lists and text input doesn't even use the mouse besides touches

File: loader/src/ui/mods/GeodeStyle.cpp [hunk 3]
```diff
@@ -217,19 +222,18 @@ std::string geodeTagName(std::string_view tag) {
...
-ListBorders* createGeodeListBorders(CCSize const& size) {
+ListBorders* createGeodeListBorders(CCSize const& size, bool forceDisableTheme) {
     auto ret = ListBorders::create();
-    if (isGeodeTheme()) {
+    const bool geodeTheme = isGeodeTheme(forceDisableTheme);
+    if (geodeTheme) {
         ret->setSpriteFrames("geode-list-top.png"_spr, "geode-list-side.png"_spr, 2);
+        ret->setContentSize(size);
+    } else {
+        ret->setContentSize(size + ccp(5, 5));
```
Comment (SMJSGaming) [inline loader/src/ui/mods/GeodeStyle.cpp:232]: To explain this real quick, blame the fact that 90% of this asset is padding

GJ_commentTop_001

File: loader/src/ui/mods/GeodeStyle.cpp [hunk 4]
```diff
@@ -74,12 +74,17 @@
     });
 }
 
-bool GeodeSquareSprite::init(CCSprite* top, bool* state) {
-    if (!CCSprite::initWithFile(isGeodeTheme() ? "GE_button_05.png"_spr : "GJ_button_01.png"))
+bool isGeodeTheme(bool forceDisableTheme) {
```
Comment (HJfod) [inline loader/src/ui/mods/GeodeStyle.cpp:77]: This should be marked `static`

File: loader/src/ui/mods/GeodeStyle.cpp (hunk 3 above)
Comment (HJfod) [reply loader/src/ui/mods/GeodeStyle.cpp:232]: We love hardcoded values for dealing with weird textures

File: loader/src/ui/nodes/TextInput.cpp (hunk 2 above)
Comment (HJfod) [reply loader/src/ui/nodes/TextInput.cpp:154]: If `setTouchEnabled` is enough to make the input non-clickable then this is fine by me

**Review by HJfod — APPROVED** [review]

File: loader/src/ui/mods/GeodeStyle.cpp (hunk 4 above)
Comment (matcool) [reply loader/src/ui/mods/GeodeStyle.cpp:77]: should it though? its in the header

**Pull request merged**